src/rsz/include/rsz/Resizer.hh

```diff
@@ -182,6 +182,19 @@ struct BufferData
 
 class OdbCallBack;
 
+// RAII mechanism to facilitate using operation-specific settings.
```

Comment:
This seems generic enough to put in utl.  You should put the method definitions in the header as well.